rawkit/raw.py

```diff
@@ -52,14 +53,30 @@ def process(self, options=None):
         libraw.libraw_unpack(self.data)
         libraw.libraw_dcraw_process(self.data)
 
-    def save(self, filename=None):
+    def save(self, filename=None, filetype='ppm'):
         """
-        Save the raw data as a new PPM or TIFF image. This method is provided
+        Save the image data as a new PPM or TIFF image. This method is provided
         for convenience, but in general export should actually be handled by
         another library (such as Wand).
 
         Keyword arguments:
-        filename -- A filename (ending in ``.ppm`` or ``.tiff``) to save.
+        filename -- A filename to save.
+        filetype -- A filetype (``ppm`` or ``tiff``).
```

Comment:
Can you tweak these to use the format like the docstring on the Raw object? It's not as nice looking here, but it makes the docs way easier to parse. Eg.

```
:param filename: the name of a raw file to load                                                                      
:type filename: :class:`basestring`                                                                                  
:returns: A raw object                                                                                               
:rtype: :class:`Raw`
```